### PR TITLE
User&ChatRoom Model and controller

### PR DESCRIPTION
**Pull request by KhaderBallout**
@maysqunaibi

### CODE REVIEW

File: database/models/user.js [hunk 1]
```diff
@@ -1,22 +1,35 @@
...
+    listOfChatRoom:{
+        type:Array
+    },
+    password:{
+        type:String
     },
     email:{
-        type:String,
-        required:true
+        type:String
```
Comment (maysqunaibi) [inline database/models/user.js:26]: There are two `type: String` ?!

File: src/components/LogIn.js [hunk 2]
```diff
@@ -2,10 +2,33 @@ import React, { Component } from 'react';
...
+        }
+
+    handleChange = e =>{
+    alert(`${this.state.username}  ${this.state.password}`)
+    e.preventDefault()
+    }    
 
     render() {
         return (
-            <form>
```
Comment (maysqunaibi) [inline src/components/LogIn.js:8]: Why is this ??

Comment (KhaderBallout) [reply src/components/LogIn.js:8]: ??

**Comment by maysqunaibi** [comment]
Thank you, Great Job

**Pull request merged**